coalib/bears/Bear.py

```diff
@@ -273,6 +279,185 @@ def log_message(self, log_message, timestamp=None, **kwargs):
     def run(self, *args, dependency_results=None, **kwargs):
         raise NotImplementedError
 
+    def parenthesis_split(self, sentence, separator=',',
+                          lparen='(', rparen=')'):
+        nb_brackets = 0
+        sentence = sentence.strip(separator)
+        final = [0]
+        try:
+            for i, c in enumerate(sentence):
+                if c == lparen:
+                    nb_brackets += 1
+                elif c == rparen:
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `autopep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmport_hkfp/coalib/bears/Bear.py
+++ b/tmp/tmport_hkfp/coalib/bears/Bear.py
@@ -410,7 +410,7 @@
                 colored_table_data=[]
                 colors = ['red', 'white', 'blue', 'yellow', 'magenta', 'green']
                 for x in final_table_data:
-                    row=[]
+                    row = []
                     for index, y in enumerate(x):
                         row.append(colored(y, colors[index]))
                     colored_table_data.append(row)
```